1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,151 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User is logged in
+        When they open the Project Wizard
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Save Button
+        And they see a Next Button
+
+    @xxx
+    Scenario Outline: User opens a Project in the Project Wizard
+        Given User has the "<Role>" role in the project's scope
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+
```

Comment:
Ser ut som dette er duplikat av `User Edits a Project in the Project Wizard` og kan fjernes?
Kanskje burde `And the project lacks an Approval of type "REK"` legges til i eksisterende scenario i stedet
```suggestion
```

Reply:
Det andre scenarioet mangler Curator i Examples også, så antar at en av de er feil?

Reply:
Tja, begge er vel litt feil - men også korrekt.
"My projects" har ikkje curator rollen - da den skal fungere likt for alle brukere. (Curator er også ein user, og kan delta i prosjekt på lik linje med andre usere).
Men det stemmer som du påpeiker at desse to scenario er lik og REK begrensningen bør ligge på begge to.
Eg skriver om "User opens a Project in the Project Wizard" til å bli "Curtator opens a Project in the Project Wizard" - for Curator kan søke opp eit prosjekt, opne landingpage  og endre på det i project wizard. Så da tar eg bor dei andre rollen og legger til ein kommentar om at dette scenario supplerer eit anna.